tests/steinberger_viscosity.prm

```diff
@@ -51,6 +51,7 @@ subsection Material model
     set Latent heat                      = false
     set Compressible                     = true
     set Reference viscosity              = 1e21
+    set Use lateral average temperature for viscosity = false
```

Comment:
Leave it at the default value 'true' if you can. If you can't just reduce the number of depth layers.